Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -269,7 +269,7 @@ bool CachedInterpreter::CheckIdle(CachedInterpreter& cached_interpreter, u32 idl
 
 bool CachedInterpreter::HandleFunctionHooking(u32 address)
 {
-  const auto result = HLE::TryReplaceFunction(address);
+  const auto result = HLE::TryReplaceFunction(address, PowerPC::CoreMode::JIT);
```

Comment:
this looks fishy, so if it's really correct, a comment would be good

Reply:
Will add a comment, but for context, If you look in JitInterface.cpp, CachedInterpreter is considered a JIT (and m_mode in PowerPC.cpp will be set accordingly for that).